### ISSUE TITLE
I want to help to port to mobile. Nerdamer works 

### ISSUE THREAD

**Issue by FBarrca** [Canna71/obsidian-mathpad#19]
Hello, 

I really like this plugin and I want to use it on my tablet, so today I decided to try to port it.
I did some tests by making a plugin that uses Nerdamer, as far as I've seen, it works on mobile no problem.

That's the only dependency I've seen while browsing the code. 
Why did you make the library only desktop? Am I missing something?

Here is the repo with the plugin I made using Nerdamer: https://github.com/FBarrca/Nerdamer-Test/releases/tag/0.0.5

**Comment by Canna71** [comment]
I had some errors when testing on mobile which I cannot recall, I should go over it again anyway, it's on my TODO list :-)
Maybe it was related to MathJax, more than to Nerdamer.

**Comment by FBarrca** [comment]
Yes, I hate how Obsidian handles MathJax! Why do we need to call finishRenderMath() ?!

After tinkering today, I have both good news and bad news:

- Good news: I've created a simple plugin that evaluates inline code in Nerdamer and displays the said code with MathJax. It works perfectly on mobile.
- Bad news: I'm afraid I might not be able to help you with the plugin. I've looked at the code, and it's quite overwhelming; it's too complicated for me. (Keep in mind, I'm a junior electrical engineer, not a software engineer.)

I'll try to recreate the inline code block system you've implemented from scratch for a plugin for myself, as it is very well-thought-out.
I'll comment on the repo in this thread once it's more refined, in case you find something that aids you with this plugin.

Kind regards
